### ISSUE TITLE
Configuration including TURN with credentials leads to the segmentation fault

### ISSUE THREAD

**Issue by manifest** [lerouxrgd/datachannel-rs#6]
The libdatachannel requres username and password for a TURN server to be encoded according to the format: "turn:USERNAME:PASSWORD@HOST:PORT". Passing such a URI to the `RtcPeerConnection::new` leads to the segmentation fault.

**Comment by paullouisageneau** [comment]
The wrapper currently uses libdatachannel v0.10.4, whereas TURN support was added in v0.11.0. Still, I'm surprised it leads to a segfault, as it should be ignored.

**Comment by lerouxrgd** [comment]
Does your string include a NULL character \0 ?

**Comment by lerouxrgd** [comment]
Actually even if it did that would be a seg fault but a regular panic.

**Comment by lerouxrgd** [comment]
@manifest do you have a small example that reproduces the bug ?

**Comment by manifest** [comment]
> @manifest do you have a small example that reproduces the bug ?

This code example crashes with the segmentation fault.
```rust
let connection_options = RtcConfig::new(&[
    "stun:stun.l.google.com:19302",
    "turn:l:p@stun.l.google.com:19302",
]);

let ltConnectionHandler = ConnectionHandler::new();
let mut ltPeer = RtcPeerConnection::new(&connection_options, ltConnectionHandler)?;
```

**Comment by manifest** [comment]
A single host example seems to work just fine
```rust
let connection_options = RtcConfig::new(&[
    "turn:l:p@stun.l.google.com:19302",
]);
```

**Comment by lerouxrgd** [comment]
Thank you for the example, I pushed a fix to the branch `renaming`, could you give it a try ?

**Comment by manifest** [comment]
It's working fine now. Thanks.

Are you going to merge `renaming` branch into the `master`?

**Comment by lerouxrgd** [comment]
Yes it is planned to be merged soon.

**Comment by lerouxrgd** [comment]
@manifest note that this has been released in `0.3.0`
